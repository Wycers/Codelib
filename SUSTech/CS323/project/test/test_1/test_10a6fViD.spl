int test_1_r20()
{
  char c = '~';
}
