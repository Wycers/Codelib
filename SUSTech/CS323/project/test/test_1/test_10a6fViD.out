Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test_1_r20
        LP
        RP
      CompSt (2)
        LC
        DefList (3)
          Def (3)
            Specifier (3)
              TYPE: char
            DecList (3)
              Dec (3)
                VarDec (3)
                  ID: c
                ASSIGN
                Exp (3)
                  CHAR: '~'
            SEMI
        RC
